YoutubeExplode/Playlists/PlaylistClient.cs

```diff
@@ -42,9 +43,10 @@ public class PlaylistClient
         var author = channelId is not null && channelTitle is not null
             ? new Author(channelId, channelTitle)
             : null;
+        //Can't get description from mix playlists
+        var description = playlistExtractor.TryGetPlaylistDescription();
```

Comment:
Defaulting to empty string is fine here, we were already doing it before in similar cases